### ISSUE TITLE
Unsupervised Cross-lingual Representation Learning at Scale

### ISSUE THREAD

**Issue by juanmc2005** [juanmc2005/phd-bib#3]
## [Unsupervised Cross-lingual Representation Learning at Scale](https://arxiv.org/abs/1911.02116)
*This paper shows that pretraining multilingual language models at scale leads to significant performance gains for a wide range of cross-lingual transfer tasks. We train a Transformer-based masked language model on one hundred languages, using more than two terabytes of filtered CommonCrawl data. Our model, dubbed XLM-R, significantly outperforms multilingual BERT (mBERT) on a variety of cross-lingual benchmarks, including +13.8% average accuracy on XNLI, +12.3% average F1 score on MLQA, and +2.1% average F1 score on NER. XLM-R performs particularly well on low-resource languages, improving 11.8% in XNLI accuracy for Swahili and 9.2% for Urdu over the previous XLM model. We also present a detailed empirical evaluation of the key factors that are required to achieve these gains, including the trade-offs between (1) positive transfer and capacity dilution and (2) the performance of high and low resource languages at scale. Finally, we show, for the first time, the possibility of multilingual modeling without sacrificing per-language performance; XLM-Ris very competitive with strong monolingual models on the GLUE and XNLI benchmarks. We will make XLM-R code, data, and models publicly available.*

### Cite this paper
```bibtex
@misc{conneau2019unsupervised,
    title={Unsupervised Cross-lingual Representation Learning at Scale},
    author={Alexis Conneau and Kartikay Khandelwal and Naman Goyal and Vishrav Chaudhary and Guillaume Wenzek and Francisco Guzmán and Edouard Grave and Myle Ott and Luke Zettlemoyer and Veselin Stoyanov},
    year={2019},
    eprint={1911.02116},
    archivePrefix={arXiv},
    primaryClass={cs.CL}
}
```

**Comment by juanmc2005** [comment]
### Intro
This paper introduces a new transformer based cross-lingual language model named XLM-R.

### Model
They train a transformer with the MLM objective (Masked Language Model, like BERT) using monolingual data (no translations) on 100 languages. The training data is extracted from CommonCrawl using existing language identification models.

### Evaluation
The evaluation of the model is done on XNLI, Named Entity Recognition, Cross Lingual Question Answering and the GLUE benchmark.

### Results
Their model achieves state of the art performance on XNLI, CoNLL-2002 and CoNLL-2003 for NER and MLQA, but not on the GLUE benchmark.

#### Curse of Multilinguality
The authors hypothesize that the fixed capacity of a model constrains the fine grained modelling of each language, particularly low resource languages, where there's a lack of training data.
They have observed that positive transfer (the effect of better modelling a low resource language with a similar high resource one) increases up to a number of training languages. From this point forward, the effect of what they call "curse of multilinguality" kicks in, and performance for low resource languages starts decreasing.

#### Sentence Piece to simplify multilingual tokenization
_"We did not observe any loss in performance for models trained with SPM when compared to models trained with language-specific pre-processing and byte-pair encoding"_

#### Multilingual models can outperform their monolingual counterparts
They observe this behaviour by comparing a cross-lingual model on 7 languages to 7 individual BERT models trained on Wikipedia and CC (Common Crawl). They found that using training data in different languages in particular boosts multilingual models performance.

#### Other
Unsurprisingly, they find that increasing model capacity and shared vocabulary size can alleviate the curse of multilinguality and increase performance on downstream tasks.
They also note that larger training time is needed, and that validation perplexity as a stopping criterion caused the model to be under-tuned.
